libs/ha/ha_common_libs_k8s.py

```diff
@@ -1256,12 +1262,9 @@ def get_config_value(pod_obj, pod_list=None):
         if not resp[0]:
             LOGGER.error("Error: Failed to copy cluster.conf to local")
             return False, resp
-        try:
-            with open(local_conf, "r", encoding="utf-8") as file_data:
-                data = yaml.safe_load(file_data)
-        except IOError as error:
-            LOGGER.error("Error: Not able to read local config file")
-            return False, error
+        conf_fd = open(local_conf, 'r')
```

Comment:
Why this removed. ? we should use **with** to open file, which will implicitly close the file.  And use of try except was also required incase of file is missing or issue with open.

Reply:
hmm, because of merge conflicts resolution, this is introduce. Corrected.